### PR TITLE
Add icon and status for tf plans that are waiting for user action

### PR DESCRIPTION
**Pull request by joshri**
Closes https://github.com/weaveworks/weave-gitops-enterprise/issues/2745

Adds case for terraform folks to know when they have to take an action on their plan. Preview of what icon looks like below:
image

### CODE REVIEW

File: ui/components/KubeStatusIndicator.tsx [hunk 1]
```diff
@@ -38,11 +39,13 @@ export function computeReady(conditions: Condition[]): ReadyType {
...
-    ) {
-      return ReadyType.Reconciling;
+    if (readyCondition.status === ReadyStatusValue.Unknown) {
+      if (readyCondition.reason === "Progressing") return ReadyType.Reconciling;
+      if (readyCondition.reason === "TerraformPlannedWithChanges")
+        return ReadyType.PendingAction;
+    }
+
+    {
     }
```
Comment (chanwit) [inline ui/components/KubeStatusIndicator.tsx:49]:
```suggestion
```

**Review by chanwit — CHANGES_REQUESTED** [review]
Thank you @joshri 
The change to the readCompute function makes sense to me.

There is an empty block which causes a lint error btw.

**Comment by chanwit** [comment]
Orange icon! approved!
image

**Review by chanwit — APPROVED** [review]
LGTM

**Comment by opudrovs** [comment]
Cool, thank you for testing it, @chanwit !

**Pull request merged**